desktop-src/appxpkg/windows-desktop-application-program.md

```diff
@@ -8,6 +8,9 @@ ms.date: 11/02/2018
 
 # Windows Desktop Application Program
 
+> [!NOTE]
+> You can access analytics data in Desktop Application Program through Insights dashboard in Partner Center. In Insights dashboard, please navigate to Windows Desktop Applications section and click on Overview to view the data. 
```

Comment:
This does not need to be a note. Just add a sentence to the paragraph below, so that it reads:

> You can get detailed telemetry data and analytics reports that let you see how your Windows desktop applications are doing through the new Windows Desktop Application Program. To view these reports, navigate to **Partner Center** > [complete path or set of links] > **Windows Desktop Applications** and click on Overview.

Reply:
Updated the PR based on the above comment.